packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/upgrade-node/upgrade-node.routing.js

```diff
@@ -0,0 +1,17 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'pci.projects.project.storages.databases.dashboard.general-information.upgrade-node',
+    {
+      url: '/upgrade-node',
+      component: 'ovhManagerPciProjectDatabaseGeneralInformationUpgradeNode',
+      resolve: {
+        breadcrumb: () => null,
```

Comment:
No breadcrumb but state is not a modal ? Why ?

Reply:
Added here: https://github.com/ovh/manager/pull/5431/commits/f7ff806628ce0db5d74b8e23a2aa9c154c27c5b0